pkgs/development/libraries/libe57format/default.nix

```diff
@@ -65,5 +65,6 @@ stdenv.mkDerivation rec {
     license = licenses.boost;
     maintainers = with maintainers; [ chpatrick nh2 ];
     platforms = platforms.linux; # because of the .so buiding in `postInstall` above
+    broken = !stdenv.isLinux;
```

Comment:
```suggestion
```
platforms is already linux only which means the package is most likely broken/not working on other platforms.